s3tests_boto3/functional/__init__.py

```diff
@@ -374,6 +374,36 @@ def get_svc_client(client_config=None, svc='s3'):
 
 bucket_counter = itertools.count(1)
 
+def get_s3_client_using_iam_creds(client_config=None):
```

Comment:
I think it would be good if we use this function in our test_sts.py file?

Reply:
I have used it, but I like your version better, once all other comment are addressed, I will rebase my commits on top of yours use the version that you have defined.

Reply:
Okay